charts/__tests__/gardener-dashboard/application/role.spec.js

```diff
@@ -0,0 +1,32 @@
+//
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
nit

```suggestion
// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
```
please also fix for the other newly introduced files